### PR TITLE
Segmentation Readme

### PR DESCRIPTION
**Pull request by Louis-Dupont**

### CODE REVIEW

File: documentation/source/Segmentation.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+from super_gradients.common.object_names import Models
+
+model = models.get(
+    model_name=Models.PP_LITE_T_SEG75,      # You can use any model listed in the Models.<Name>
+    arch_params={"use_aux_heads": False},
+    num_classes=1,                          # Change this if you work on another dataset with more classes
+    pretrained_weights="cityscapes"         # Drop this line to train from scratch
+)
+```
+Notes:
```
Comment (ofrimasad) [inline documentation/source/Segmentation.md:63]: please add a comment here: model is a torch.nn.Module

File: documentation/source/Segmentation.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+    "greater_metric_to_watch_is_better": True,
+    "train_metrics_list": [BinaryIOU()],
+    "valid_metrics_list": [BinaryIOU()],
+    "loss_logging_items_names": ["loss"],
+}
+```
+
+
+### Launch Training
+Now you only need to setup a Trainer and launch the training!
```
Comment (ofrimasad) [inline documentation/source/Segmentation.md:97]: This might be a good place to shortly discuss the Trainer concept, which is not native to torch.
"The Trainer is a class that...."  
"In the example below, we will call the `train()` function, which is the main entry point for any training process... "

File: documentation/source/Segmentation.md [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+mask = torch.sigmoid(mask).gt(0.5).squeeze()
+mask = ToPILImage()(mask.float())
+display(mask)
+```
+![segmentation_prediction.png](segmentation_prediction.png)
+
+
+## Going further
+### How to launch on multiple GPUs (DDP) ?
+To run the Training using Distributed Data Parallel (DDP), all you need to do is to call a magic function `setup_device` before instantiating the Trainer.
```
Comment (ofrimasad) [inline documentation/source/Segmentation.md:140]: Instead, add references to other pages (DDP, using recipes...)

**Review by ofrimasad — COMMENTED** [review]
Looks great!
a few minor comments inline

File: documentation/source/Segmentation.md (hunk 1 above)
Comment (Louis-Dupont) [reply documentation/source/Segmentation.md:63]: Changed in d1fb7168b1580600f5594a42819fa8246042c207

File: documentation/source/Segmentation.md (hunk 2 above)
Comment (Louis-Dupont) [reply documentation/source/Segmentation.md]: Changed in d1fb7168b1580600f5594a42819fa8246042c207

File: documentation/source/Segmentation.md (hunk 3 above)
Comment (Louis-Dupont) [reply documentation/source/Segmentation.md]: Changed in d1fb7168b1580600f5594a42819fa8246042c207
It's just a placeholder for now, I have a PR open to add explanation on how to use DDP (https://github.com/Deci-AI/super-gradients/pull/662)

**Review by ofrimasad — APPROVED** [review]
LGTM
